cmd/cdi-operator/BUILD.bazel

```diff
@@ -8,14 +8,14 @@ go_library(
     deps = [
         "//pkg/operator/controller:go_default_library",
         "//pkg/util:go_default_library",
-        "//staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1:go_default_library",
```

Comment:
correct me if im wrong, this is just about consistency, pulling the API lib similarly to how external projects would?

Reply:
Curious about this as well.  Lot of references were moved from //staging to //vendor.  Was this done manually aor via `make deps-update`?

Reply:
definitely not a `make deps-update` since I did that in https://github.com/kubevirt/containerized-data-importer/pull/2632

Reply:
added the line back.